ui/app/AppLayouts/Wallet/views/LeftTabView.qml

```diff
@@ -321,6 +279,90 @@ Rectangle {
 
             readonly property bool footerOverlayed: contentHeight > availableHeight
 
+            header: AbstractButton {
```

Comment:
Control would have been just fine, I am not sure if AbstractButton if this is helping you in anyway?
Also can we move this to separate qml file under controls? its become very long.

Reply:
Seconded

Reply:
Switch from abstract button to control. However when I extract to a component, thing just don't work in terms of layout
Also parent.ListView.view becomes undefined for no obvious reason to me